### ISSUE TITLE
TinyMCE 5: Spellchecker contextmenu could be more clear

### ISSUE THREAD

**Issue by alecpl** [tinymce/tinymce#4789]
I think spelling suggestions should be somehow separated from the Add to Dictionary, Ignore and Ignore All options. Also the left margin looks strange here.
![spell](https://user-images.githubusercontent.com/546788/51630910-94256e00-1f4b-11e9-9288-4ab08c1e8d3a.png)

**Comment by TheSpyder** [comment]
Oh, oops. I know why that happened; should be an easy fix.

**Comment by alecpl** [comment]
BTW, I see some differences in how contextmenu works in v5 and v4. In v4 if I click on a mispelled word I don't see Link option, only spellchecker options. Also, If I click on text in v4 I see Link, Image, Table options, while in v5 only Link.

**Comment by TheSpyder** [comment]
Yes, the v5 context menu shows a dynamic list based on the cursor context. The v4 menu was static and options just enabled and disabled as the cursor moved around.
